### ISSUE TITLE
Export JSON without schema and export Avro without schema registry

### ISSUE THREAD

**Issue by darrenhaken** [confluentinc/kafka-connect-storage-cloud#86]
I'm trying to use the connector with vanilla Kafka. Is it possible to use this without the schema registry for Avro?

Also, I can't see examples on the docs to work with JSON, can you provide any?

**Comment by justinsoong** [comment]
I'm trying to do the same, running into Dependency errors around AvroDataConverter, in my case i have a schema registry running, and also including ../share/java in the plugin path for a standard c4.0 distribution, did you have any luck? @darrenhaken

**Comment by justinsoong** [comment]
@darrenhaken i=I got it to work
either in your worker config, or your new connector config
```
key.converter.schemas.enable=false
value.converter.schemas.enable=false
key.converter=org.apache.kafka.connect.storage.StringConverter
value.converter=org.apache.kafka.connect.json.JsonConverter
```

**Comment by farmdawgnation** [comment]
I actually ran into the same issue (wanting to use this connector with Avro data without needing the registry). I built a thing that should let you do it by specifying a different converter: https://github.com/farmdawgnation/registryless-avro-converter

We're using it in production without any issues so far. It currently requires a reader schema be provided, which I'm working on making optional.
